scripts/esbuild/util.ts

```diff
@@ -112,5 +112,15 @@ export function getBaseEsbuildOptions(): ESBuildOptions {
     bundle: true,
     legalComments: 'inline',
     logLevel: 'info',
+    target: getEsbuildTargets(),
   };
 }
+
+/**
+ * Get build targets with minimal supported browser versions
+ * @see https://stenciljs.com/docs/support-policy#browser-support
+ * @returns an array of build targets
+ */
+export function getEsbuildTargets(): string[] {
+  return ['node16', 'chrome79', 'edge79', 'firefox70', 'safari14'];
```

Comment:
My intention is to have it streamlined across all builds to add more clarity. While the runtime for the compiler is obvious it might not be for other parts, e.g. testing. I don't believe adding browser build targets to outputs that run in Node.js only has a big effect if any at all as we aren't using any WebAPIs in the source anyway.

I don't have a strong opinion on this one and happy to revert it if we prefer to keep specific targets for specific outputs.